consensus/polybft/contractsapi/helper.go

```diff
@@ -26,6 +26,14 @@ type EventAbi interface {
 	ParseLog(log *ethgo.Log) (bool, error)
 }
 
+// StructAbi is an interface representing a structure in contractsapi
+type StructAbi interface {
```

Comment:
I don't think we need this interface anymore, since it is exactly the same as `ABIEncoder`.